dusty/commands/upgrade.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+import shutil
+from subprocess import check_output, check_call, CalledProcessError
+import urllib
+
+from .. import constants
+from ..daemon import close_client_connection
+from ..log import log_to_client
+from ..payload import daemon_command
+
+
+def _get_latest_version():
+    """Gets latest Dusty binary version using the GitHub api"""
+    url = 'https://api.github.com/repos/gamechanger/dusty/releases/latest'
+    conn = urllib.urlopen(url)
+    json_data = conn.read()
+    return json.loads(json_data)['tag_name']
```

Comment:
This function seems pretty brittle, you're not checking the HTTP status code or anything